addons/mail/static/tests/tours/discuss_channel_public_tour.js

```diff
@@ -66,6 +66,39 @@ registry.category("web_tour.tours").add("discuss_channel_public_tour.js", {
             content: "Check message contains the attachment",
             trigger: '.o-mail-Message .o-mail-AttachmentCard:contains("text.txt")',
         },
+        {
+            trigger: ".o-mail-Message [title='Add a Reaction']",
+            run: "click",
+        },
+        {
+            trigger: ".o-EmojiPicker",
+        },
+        {
+            trigger: ".o-Emoji:contains('🙂')",
+            run: "click",
+        },
```

Comment:
`.o-EmojiPicker` is probably useless or should be merged in the next step if we want a more specific selector